lib/serverengine/daemon.rb

```diff
@@ -47,14 +47,23 @@ def initialize(server_module, worker_module, load_config_proc={}, &block)
       @chuser = @config[:chuser]
       @chgroup = @config[:chgroup]
       @chumask = @config[:chumask]
+
+      @pid = nil
+      @command_pipe = @config.fetch(:command_pipe, nil)
+      @command_sender = @config.fetch(:command_sender, "signal")
```

Comment:
Signals don't work on Windows at all, is that right?
If yes, this requires all server application developers to set "pipe" if the app is running on Windows. But most of users don't want to care about the execution platform and write the same code (I believe). So it sounds better that here uses `"pipe"` as the default value on Windows.